types/mithril-global/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for non-npm package Mithril 2.0
+// Type definitions for non-npm package Mithril 2.2
 // Project: https://mithril.js.org/
 // Definitions by: Mike Linkovich <https://github.com/spacejack>
 //                 Claudia Meadows <https://github.com/dead-claudia>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.2
+// TypeScript Version: 5.3
```

Comment:
```suggestion
```

Reply:
This is moving everyone using this package to typescript version 5.3 which is not recommended. 
Please resolve errors such that its not TS version dependent and remove this line.

Reply:
Found a way to do this! If tests don't fail we're good to go!